.buildkite/pipeline.full.yml

```diff
@@ -198,6 +198,8 @@ steps:
           - "--fail-fast"
     concurrency: 9
     concurrency_group: 'browserstack-app'
+    soft_fail:
```

Comment:
Although we initially saw problems on Android 4, I don't think there is an inherent infrastructure issue there that we're working around temporarily.  As it represents the minimum supported version, I'd rather not soft fail unless we really have to.